pygame_life.py

```diff
@@ -45,21 +134,18 @@ def update_grid(grid: Grid) -> Grid:
         new_cells.add((pos[0], pos[1]))
 
     return Grid(grid.dim, new_cells)
+    

```

Comment:
Maybe these new lines aren't needed here? Unless you think the previous code was unreadable without them 😄